### ISSUE TITLE
Keyboard.removeListener is not a function in RN 0.65

### ISSUE THREAD

**Issue by Aure77** [facebook/react-native#32313]
Keyboard.removeListener is marked as "deprecated" in documentation but crash at runtime.

## Description
As mentioned in documentation:
https://reactnative.dev/docs/keyboard#removelistener
Keyboard.removeListener is marked as "deprecated" (not removed). But a runtime, I got this error:
`TypeError: _reactNative.Keyboard.removeListener is not a function. (In '_reactNative.Keyboard.removeListener('keyboardDidShow', this._handleKeyboardShow)', '_reactNative.Keyboard.removeListener' is undefined)`

So third-party libraries based on  `Keyboard.removeListener` are failing using react-native 0.65+
I know how to fix the problem. But documentation is not clear about availability of this method within `Keyboard` class.

## React Native version:
```
info Fetching system and libraries information...
System:
    OS: macOS 11.5.2
    CPU: (4) x64 Intel(R) Core(TM) i7-7660U CPU @ 2.50GHz
    Memory: 369.01 MB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 14.16.1 - ~/.nvm/versions/node/v14.16.1/bin/node
    Yarn: 1.22.11 - /usr/local/bin/yarn
    npm: 6.14.12 - ~/.nvm/versions/node/v14.16.1/bin/npm
    Watchman: Not Found
  Managers:
    CocoaPods: 1.10.1 - /Users/me/.rbenv/shims/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 14.5, DriverKit 20.4, macOS 11.3, tvOS 14.5, watchOS 7.4
    Android SDK: Not Found
  IDEs:
    Android Studio: 4.2 AI-202.7660.26.42.7351085
    Xcode: 12.5/12E262 - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.9 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.2 => 17.0.2
    react-native: 0.65.1 => 0.65.1
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```

## Steps To Reproduce
```
Keyboard.addListener("keyboardWillShow", handleKeyboardDidShow);
Keyboard.removeListener("keyboardWillShow", handleKeyboardDidShow); // error at runtime
console.warn(Keyboard.removeListener); // undefined
```

## Expected Results
Documentation must be updated (method removed) or `Keyboard.removeListener` method should be defined as previous version (with a deprecated log message for example)

**Comment by mozadeh** [comment]
@L3au I had the same problem 0.66 and I changed the react native code by fixing the function name to `removeListener` and can confirm it prevented crashes. I'm sure it's a mistake in the code because a few lines before that is `addListener` not `addEventListener`

**Comment by lachenmayer** [comment]
This issue is fixed in 0.66.3. https://github.com/facebook/react-native/commit/8880c09076e4727768ace26a74766cbe6f64021c

**Comment by cortinico** [comment]
Closing as this version of React Native is several years old. Please re-open a new issue against the latest stable if the issue persists
